views/404.ejs

```diff
@@ -1 +1,16 @@
-<h1>Sorry, this page cannot be found</h1>
\ No newline at end of file
+<div class="container">
+    <h1>Oops!</h1>
+    <img class="image" src="/images/dog.jpg" alt="cartoon dog">
+    <h2>The page you were looking for doesn't appear to be here, let us know who you think should be fired!</h2>
+    <h2>Or we'll blame it on the ifElse dog!</h2>
+    <h4>Error code: 404</h6>
+    <p>Try one of these links instead:</p>
+    <ul>
+        <li><a href="/">Home</a></li>
+        <li><a href="/about">About</a></li>
+        <li><a href="/blogs">Blogs</a></li>
+        <li><a href="/contact">Contact</a></li>
+        <li><a href="/login">Login</a></li>
+        <li><a href="/signup">Signup</a></li>
+    </ul>
```

Comment:
Thanks for this Chris! :) I don't think you need this ul here as the main nav will still be visible and accessible on the left of the screen.